src/Microsoft.AspNet.Session/SessionServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Session;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class SessionServiceCollectionExtensions
+    {
+        public static IServiceCollection AddSession([NotNull] this IServiceCollection services)
+        {
+            return services.AddSession(configure: null);
+        }
+
+        public static IServiceCollection AddSession([NotNull] this IServiceCollection services, Action<SessionOptions> configure)
```

Comment:
hmm. I'm calling this from the other `AddSession` overload. That's why I allowed null. But anyway I check for null inside and do accordingly.